ods_ci/tests/Resources/CLI/ModelServing/llm.resource

```diff
@@ -745,21 +746,25 @@ Create PVC And Download Model From S3
     Create File From Template    ${DOWNLOAD_PVC_FILEPATH}    ${DOWNLOAD_PVC_FILLED_FILEPATH}
     ${rc}    ${out}=    Run And Return Rc And Output    oc -n ${namespace} apply -f ${DOWNLOAD_PVC_FILLED_FILEPATH}
     Should Be Equal As Integers    ${rc}    ${0}
-    Wait For Pods To Be Ready    label_selector=name=download-${model_name}    namespace=${namespace}
+    Run Keyword And Ignore Error   Wait For Pods To Be Ready    label_selector=name=download-${model_name}    namespace=${namespace}
     Wait For Pods To Succeed    label_selector=name=download-${model_name}    namespace=${namespace}
     ...    timeout=${download_timeout}
     ${rc}    ${out}=    Run And Return Rc And Output    oc delete pods -l name=download-${model_name} -n ${namespace}
     Should Be Equal As Integers    ${rc}    ${0}
 
 Extract Service Port
     [Arguments]    ${service_name}    ${namespace}    ${protocol}
-    IF    "${protocol}" == "grpc"
+    ${name}     Set Variable    name
```

Comment:
Can you use a different name for this variable? it's confusing

Reply:
what do u suggest ?

Reply:
perhaps something like `protocol_name`, `jsonpath_name`, `jsonpath_parent`. As long as the variable isn't called `name` and its value is `name` it would be better.

Reply:
changed to protocol_name